pkg/edgeview/src/edge-view-init.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+if [ -f "/config/remote_access_disabled" ]; then
+  echo "Remote access is disabled"
+  exit 1
```

Comment:
Does this cause the container to fail to start?
If so, is there any fallout from this from EVE's perspective? Or does everything else stay happy?

And if the user tries to enable edge-view from zedcloud is there something in zedagent which will return or log some warning/error?

Reply:
@shjala I think a better place to block this is inside https://github.com/lf-edge/eve/blob/master/pkg/pillar/cmd/zedagent/parseedgeview.go#L24, if the config '/config/remote_access_disabled' exist, log a Notice, and call 'removeEvFiles()' to remove any edge-view configuration. Then the edgeview container is running, but no edge-view processes will be started. Otherwise, if jump out of edgeview inside the process itself, it later finds the /run/edgeview configuration, it will start the process again. etc.

Reply:
@naiming-zededa thanks for the hint, I will change it accordingly.